.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
```

Comment:
https://user-images.githubusercontent.com/38065883/172700445-fb103f24-94dc-4754-b969-1c23296633d2.mov

Still something strange is happening if you delete one card. Layout shouldn't get broken if amount of cards has changed